changelog/2.072.0_pre.dd

```diff
@@ -124,30 +124,32 @@ $(HR)
 $(BUGSTITLE Language Changes,
     $(LI $(LNAME2 deprecated_commaexp, Using the result of a comma expression is deprecated.)
 
-    $(P Comma expressions expressions have proven to be a frequent source of confusion, and bug.
-        Using their result will now trigger a deprecation message.)
+    $(P Comma expressions have proven to be a frequent source of confusion, and bugs -
+    see $(LINK2 https://issues.dlang.org/show_bug.cgi?id=2659#c2, here) for examples.)
```

Comment:
> Use HTTPS instead of LINK2

Or don't. HTTPS doesn't buy us anything. This is getting old. I'm a little disappointed that you just keep pushing for it, @andralex, seemingly without reason. ([previously](https://github.com/dlang/phobos/pull/4411#issuecomment-226392170))